action.yml

```diff
@@ -3,6 +3,11 @@ description: '🐶 Run rubocop with reviewdog on pull requests to improve code r
 author: 'mgrachev (reviewdog)'
 inputs:
   # Please maintain inputs in alphabetical order
+  fail_level:
+    description: |
+      Exit code 1 for reviewdog if it finds at least 1 issue with severity greater than or equal to the given level [none,any,info,warning,error].
+      Default is `none`.
+    default: 'none'
   fail_on_error:
```

Comment:
https://docs.github.com/ja/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddeprecationmessage

It is better to add `deprecationMessage` to notify users that this input is closing down and mention `fail_level` as an alternative.